iceoryx_posh/include/iceoryx_posh/internal/runtime/ipc_interface_creator.hpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2019, 2021 by Robert Bosch GmbH, Apex.AI Inc. All rights reserved.
```

Comment:
Content of this file was moved, hence the header is correct despite the file being a new one